src/toggle/toggle.js

```diff
@@ -2,21 +2,40 @@ import React, {Component} from "react"
 import * as stateChangeTypes from "./stateChangeTypes"
 
 class Toggle extends Component {
-	state = {on: true}
+	static defaultProps = {
+		stateReducer: (state, changes) => changes,
+	}
+
+	state = {on: false}
+
+	internalSetState = () => {
+		// we want to suggest the changes to be made
+		// and allow the user to decide if the type of change to be performed
+		// is one they want made
+		// we'll make our changes
+		// and the any changes the user wants to make
+		//
+	}
 
-	toggle = () => {
-		this.setState(({on}) => ({on: !on}))
+	toggle = cb => {
+		this.internalSetState()
 	}
 
-	getToggleProps = () => {
+	getToggleProps = ({onKeyDown, ...rest} = {}) => {
 		return {
 			role: "switch",
+			tabIndex: 0,
```

Comment:
The value of `tabIndex: 0` should be tabindex if we intend to use as a prop on an element

Reply:
I understand now. Referring to this [keyboard accessibility, tabindex](https://webaim.org/techniques/keyboard/tabindex), I see what changes we should make -> `tabindex:0`

Reply:
The issue here isn't that we have the `tabindex` label wrong - it's that we're not assigning a value to `tabIndex`. Refer to [1cf0696](https://github.com/krfong916/swish-components/pull/1/commits/1cf0696e6d1b4c588d3f5f3c8c170afa6001e7a1) for the changes we're looking for.